packaging/apple/mac_packager.sh.in

```diff
@@ -40,10 +40,10 @@ cd @PROJECT_BINARY_DIR@
 gunzip -f MedinriaPackage.sparseimage.gz
 
 devName=`hdiutil attach -readwrite -noverify -noautoopen MedinriaPackage.sparseimage | egrep '^/dev/' | sed 1q | awk '{print $1}'`
-diskutil rename "medInria base" "medInria @MEDINRIA_SUPERBUILD_VERSION@"
+diskutil rename "medInria base" "MUSIC @MEDINRIA_SUPERBUILD_VERSION@"
 \cp -rf @medInria_BINARY_DIR@/bin/TmpInstall/medInria.app /Volumes/"medInria @MEDINRIA_SUPERBUILD_VERSION@"/
```

Comment:
to replace the line 44 with 
**cp -rf @medInria_BINARY_DIR@/bin/TmpInstall/MUSIC.app /Volumes/"MUSIC @MEDINRIA_SUPERBUILD_VERSION@"** in order to the mac package generation to work.